packages/devextreme/scss/widgets/generic/gridBase/_index.scss

```diff
@@ -390,14 +390,6 @@ $grid-masterdetail-padding: 20px;
       .dx-command-edit {
         width: $generic-grid-base-command-edit-column-width;
         min-width: $generic-grid-base-command-edit-column-width;
-
```

Comment:
We tested it together with @EugeniyKiyashko and decided to roll back these changes.
Because in the dxDataGrid we've our button implementation, and without these styles, disabled buttons render incorrectly.